Mathlib/RingTheory/Ideal/Span.lean

```diff
@@ -235,6 +235,13 @@ theorem span_singleton_neg (x : α) : (span {-x} : Ideal α) = span {x} := by
   simp only [mem_span_singleton']
   exact ⟨fun ⟨y, h⟩ => ⟨-y, h ▸ neg_mul_comm y x⟩, fun ⟨y, h⟩ => ⟨-y, h ▸ neg_mul_neg y x⟩⟩
 
+@[simp]
+theorem span_singleton_abs [LinearOrder α] (x : α) :
+    span {|x|} = span {x} := by
+  obtain h | h := abs_choice x
```

Comment:
Untested, but I think something like
```lean
obtain h | h := abs_choice x <;>
simpa [h]
```
works (maybe `simpa using h`).

Reply:
I think it is true but it's probably harder to use and I cannot see any real use for it.